packages/smart-account/src/SmartAccount.ts

```diff
@@ -496,42 +496,27 @@ class SmartAccount extends EventEmitter {
       config: state,
       context: this.getSmartAccountContext(chainId)
     }
-    const gasLimit = transactionDto.transaction.gasLimit
```

Comment:
Is it necessary to remove this? it's only removed for fallback relayer or before sending it to any relayer?